streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingKeyValueStoreTest.java

```diff
@@ -173,7 +173,7 @@ public void shouldQueryRange() throws Exception {
         store.put(hello, world);
         store.put(hi, there);
         store.put("zooom", "home");
-        final KeyValueIterator<String, String> range = store.range(hello, "zooom");
+        final KeyValueIterator<String, String> range = store.range(hello, hi);
```

Comment:
I am still puzzled why the test did not fail with "zooom" as end range. Can you explain? \cc @dguy WDYT?